static/styles.css

```diff
@@ -181,21 +148,109 @@ a {
   }
 }
 
+.grid-wrapper {
+  -webkit-mask-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0) 0%,
+    rgba(0, 0, 0, 0) 20%,
+    rgba(0, 0, 0, 1) 100%
+  );
+  mask-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0) 0%,
+    rgba(0, 0, 0, 0) 20%,
+    rgba(0, 0, 0, 1) 100%
+  );
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  left: 0;
+  top: 30%;
+  left: 0;
+  right: 0;
+}
+
+.grid {
+  position: absolute;
+  width: 200%;
+  top: -20%;
+  left: -50%;
+  height: 1200px;
+  background-image: repeating-linear-gradient(
+      0deg,
+      #5e4cce,
+      #5e4cce 2px,
+      transparent 2px,
+      transparent 40px
+    ),
+    repeating-linear-gradient(
+      -90deg,
+      #5e4cce,
+      #5e4cce 2px,
+      transparent 2px,
+      transparent 50px
+    );
+  background-size: 50px 40px;
+  animation: fly 0.4s linear;
+  animation-iteration-count: infinite;
+}
+
+@keyframes fly {
+  0% {
+    transform: perspective(800px) rotateX(70deg) translateY(0%);
+  }
+  100% {
+    transform: perspective(800px) rotateX(70deg) translateY(40px);
+  }
+}
+
+@keyframes fadeInAndGrow {
+  0% {
+    transform: scale(0.7);
+    opacity: 0;
+  }
+
+  50% {
+    opacity: 1;
+  }
+
+  100% {
+    transform: scale(1);
+    opacity: 1;
+  }
+}
+
+.skyline {
```

Comment:
This is cool! It's broken though, at least on laptop viewports.

* Make responsive